lobby-db-dao/src/main/java/org/triplea/lobby/server/db/dao/BadWordsDao.java

```diff
@@ -18,8 +18,6 @@
   int removeBadWord(@Bind("word") String badWordToRemove);
 
   @SqlQuery(
-      "select count(bw.word) "
-          + "from bad_word bw "
-          + "where lower(:word) like '%' || lower(bw.word) || '%'")
-  int containsBadWord(@Bind("word") String word);
+      "select exists (select * from bad_word where lower(:word) like '%' || lower(word) || '%')")
```

Comment:
The power of SQL

Reply:
FWIW, it's actually a significant improvement to use `exists` over `count`:
https://blog.jooq.org/2016/09/14/avoid-using-count-in-sql-when-you-could-use-exists/

Runtime performance is:

> 40x on PostgreSQL

Reply:
Not surprising to see a performance improvement since postgres now no longer has to go through all entries to get the exact number, but instead can abort once it finds a single entry.
Factor 40 is a lot though